scripts/spark_eval/README.md

```diff
@@ -1,69 +1,121 @@
 # Spark Evaluation
 
-This directory contains scripts for testing Spark command lines. It is based on this [test definition document](https://docs.google.com/document/d/1OEfV2XNXdbGQQdW-gRaQYY2QRgGsWHUNKJUSAj3qFlE/edit), but also has scripts for running full pipelines.
+This directory contains scripts for testing GATK pipelines on Spark - either on a dedicated cluster or on Google Cloud Dataproc.
+
+## TL;DR
+
+```bash
+export API_KEY=...
+export GCS_CLUSTER=...
+
+# Sanity check on small data (a few mins)
+./run_gcs_cluster.sh small_reads-pipeline_gcs.sh
+
+# Run on exome (<1hr)
+nohup ./run_gcs_cluster.sh exome_reads-pipeline_gcs.sh &
+
+# Run on genome (a few hrs)
+nohup ./run_gcs_cluster.sh genome_md-bqsr-hc_gcs.sh &
+
+# Check results
+cat results/*
+```
 
 ## Obtaining and preparing the data
 
-Most of the data can be obtained from the [GATK resource bundle](https://software.broadinstitute.org/gatk/download/bundle).
+There are three main datasets of increasing size: _small_ , _exome_, and _genome_ (WGS). The _small_ data is useful for sanity checking command lines before running them on the larger _exome_ and whole _genome_ datasets.
 
-There is also some data in a GCS bucket for this evaluation: _gs://hellbender/q4_spark_eval/_.
+The datasets are stored in GCS buckets, so if you run using GCS input and output then there is no initial data preparation.
 
 To copy the exome data into the cluster run:
 
 ```bash
-./prep_data_exome_gcs.sh
+./exome_copy_hdfs.sh
 ```
 
 For the whole genome data, run:
 
 ```bash
-./prep_data_genome_gcs.sh
+./genome_copy_hdfs.sh
```

Comment:
these names don't seem to match the names of the copy scripts.  I see `copy_genome_to_hdfs.sh`

Reply:
Fixed